packages/typespec-ts/test/unit/bytesGenerator.spec.ts

```diff
@@ -167,6 +172,7 @@ describe("bytes", () => {
         const parameters = await emitParameterFromTypeSpec(
           `
               @route("/uploadFile")
+              #suppress "deprecated" "For test"
```

Comment:
refer to : https://github.com/Azure/typespec-azure/pull/2397/files#diff-6fd39508d36aff27efd8f480135d7d59db6e7d714df10439342a4d21a6781f0eR33